utils/setup.py

```diff
@@ -2,11 +2,11 @@
 from getpass import getpass
 
 
-def setup():
+def setup(path="."):
```

Comment:
should this accept default values?
also probably should change the argument name to `config_path` to be more precise